tutorials/assets_pipeline/importing_3d_scenes/available_formats.rst

```diff
@@ -0,0 +1,162 @@
+.. _doc_importing_3d_scenes_available_formats:
+
+Available 3D formats
+====================
+
+When dealing with 3D assets, Godot has a flexible and configurable importer.
+
+Godot works with *scenes*. This means that the entire scene being worked on in
+your favorite 3D modeling software will be transferred as close as possible.
+
+Godot supports the following 3D *scene file formats*:
+
+- glTF 2.0 **(recommended)**. Godot has full support for both text (``.gltf``)
+  and binary (``.glb``) formats.
+- ``.blend`` (Blender). This works by calling Blender to export to glTF in a
+  transparent manner (requires Blender to be installed).
+- DAE (COLLADA), an older format that is fully supported.
```

Comment:
DAE (COLLADA), an older format that is fully supported.

There are open bugs regarding skeleton import, but we aren't able to deprecate Collada importer.

Reply:
That's the exact same text as currently in the docs.

Reply:
COLLADA is basically deprecated and has known issues, so support is poor. I would at a minimum remove the word "fully" since it is certainly not fully supported, 
The biggest support issue is that skinned meshes are currently completely broken. See #68410.